differ/storage.go

```diff
@@ -481,7 +482,7 @@ func (storage DBStorage) WriteNotificationRecord(
 		notificationRecord.AccountNumber, notificationRecord.ClusterName,
 		notificationRecord.NotificationTypeID, notificationRecord.StateID,
 		notificationRecord.Report, notificationRecord.UpdatedAt,
-		notificationRecord.NotifiedAt, notificationRecord.ErrorLog)
+		notificationRecord.NotifiedAt, notificationRecord.ErrorLog, 1)
```

Comment:
maybe we should use a type instead of a hardcoded int? f.e `NotificationBackendEventTypeID`

Reply:
I'll use a constant